src/Entity.php

```diff
@@ -48,10 +48,20 @@ protected function getEntityUrl()
         $fullClassName = get_class($this);
         $pos = strrpos($fullClassName, '\\');
         $className = substr($fullClassName, $pos + 1);
-        $className = lcfirst($className);
+        $className = $this->snake_case($className);
         return $className.'s/';
     }
 
+    protected function snake_case($input, $delimiter = '_')
```

Comment:
Snake case has a fixed delimited by definition, I think that param can be removed.

Reply:
Laravel `snake_case` function definition had it.

Reply:
Should I still remove it?

Reply:
Yes, I think it can be removed till we need it.

Reply:
ok, removing